BB-SC/MWLayer/Configuration/BB_Local_Update_Manager.md

```diff
@@ -0,0 +1,76 @@
+
+# Local Update Manager
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Configuration
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+A Local update manager is a local instance of an (OTA) Update master. It is meant to be deployed and operated on distributed vehicle network nodes. It must provide a similar, but limited functionality compared to the OTA master. It shall be able to manage locally backup storage and rollback of software controlled by the OTA Master. The local update manager may become necessary as the OTA master will not or just with complex implementations (which may interfere with the network capabilities and the overall in-vehicle security) be able to deploy SW updates across the vehicle network within a zonal architecture. The Local update manager will deploy/install software in a dedicated local area of the vehicle network under control of the update master. It will ensure also local API for memory control and security services control.
+
```

Comment:
Depending on which local context this applies to, there are existing implementations for this.
- AUTOSAR: UCS - but not OSS-usable
- higher-level: Eclipse Ankaios, Eclipse Kanto, Eclipse Symphony, Eclipse BlueChi

Reply:
Agreed. Will be taken care of in the next pull request. will be linked in Known Implementations